nodes/modifier_change/opensubdivide.py

```diff
@@ -0,0 +1,111 @@
+# ##### BEGIN GPL LICENSE BLOCK #####
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software Foundation,
+#  Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+#
+# ##### END GPL LICENSE BLOCK #####
+
+import bpy
+from bpy.props import IntProperty
+from sverchok.node_tree import SverchCustomTreeNode
+from sverchok.data_structure import updateNode, match_long_repeat
+
+enable_module = False
+try:
+    import pyOpenSubdiv
+    from pyOpenSubdiv.pysubdivision import pysubdivide    
+    enable_module = True
+except ModuleNotFoundError:
+    enable_module = False
+
+from itertools import chain 
+import traceback 
+class SvOpenSubdivideNode(bpy.types.Node,SverchCustomTreeNode):
+    bl_idname = "SvOpenSubdivideNode"
+    bl_label = "OpenSubdiv"
+    bl_icon = 'OUTLINER_OB_EMPTY'
+    sv_icon = None 
+    
+    maxSubdivision = 5 # Creates a self.maxSubdivision attribute
+    maxlevel : IntProperty(name='level',default=0,min=0,max=maxSubdivision,update=updateNode)
+
+    # Mute Node Implementation 
+    @property
+    def sv_internal_links(self):
+        mapping =  [
+            (self.inputs['Vertices'],self.outputs['Vertices']),
+            (self.inputs['Edges'],self.outputs['Edges']),
+            (self.inputs['Faces'],self.outputs['Faces'])
+        ]        
+        return mapping 
+
+    def sv_init(self,context):        
+        self.inputs.new('SvVerticesSocket', "Vertices")
+        self.inputs.new('SvStringsSocket',"Edges")
+        self.inputs.new('SvStringsSocket', "Faces")
+
+        socket = self.inputs.new('SvStringsSocket', "Levels")
+        socket.use_prop=True
+        socket.prop_name = 'maxlevel'
+        socket.default_property_type = 'int'
+        socket.default_int_property = 0 
+        socket.int_range = (0,self.maxSubdivision) # This does not actually appear to limit the subdivision levels 
+
+        self.outputs.new('SvVerticesSocket', "Vertices")
+        self.outputs.new('SvStringsSocket', "Edges")
+        self.outputs.new('SvStringsSocket', "Faces")
```

Comment:
```suggestion
    def sv_init(self,context):        
        self.inputs.new('SvVerticesSocket', "Vertices")
        self.inputs.new('SvStringsSocket', "Faces")

        socket = self.inputs.new('SvStringsSocket', "Levels")
        socket.use_prop=True
        socket.default_property_type = 'int'
        socket.default_int_property = 0 

        self.outputs.new('SvVerticesSocket', "Vertices")
        self.outputs.new('SvStringsSocket', "Edges")
        self.outputs.new('SvStringsSocket', "Faces")
```
You should use either `use_prop` or `prop_name`. In the first case property of the socket will be used in the second property of the node of the socket. In first case you can't set maximum limit but in secot reading the property takes O(len(tree.nodes))